index.html

```diff
@@ -179,6 +185,39 @@ <h2>CSS</h2></li>
             </ul>
            </div>
         </div>
+        <div class="contact_section">
+            <div class="contact_container">
+                <h1 class="contactform-header">Contact me</h1>
+                <p>If you have an apllication you
+                    are interested in developing, a 
+                    feature that you need built or 
+                    a project that needs coding.gitn
+                    I'd love to help with it.
+                </p>
+                <form class="contact_form__form" action="https://formspree.io/f/mbjbnawr" method="POST" id="form">
+                    <ul class="form">
+                        <li>
+                            <label for="">
+                                <input class="input-text" type="text" name="name" maxlength="30" placeholder="lorem ipsum" required>
+                              </label>
+                        </li>
+                        <li>
+                            <label for="">
+                                <input class="input-text" type="text" name="e-mail" placeholder="loremipsum@gmail.com" id="email" required>
+                            </label>
+                        </li>
+                        <li>
+                            <label for="">
+                                <input class="message" type="text" id="message" name="Full Name" col="30" row="10" maxlength="500" placeholder="Write your mesage here...">
+                            </label><br>
+                        </li>
+                    </ul>
+                   <button class="btn-action" id="form-btn">Get in touch</button><br><br><br>
+                </form>
+  
+               
+           </div>
+           </div>
```

Comment:
- You have done well so far,  but according to the project requirements, you are required to use the `flexbox` display property to align the content of this section. Please fix this 🙏

Reply:
Changes in applying flex to my form have been applied to line 389 of styles.css with display flex, flex-direction column, and align content at the starting position.

Reply:
Thank you for this correction.

Reply:
Good job by making your form flexbox but the idea is to make the section flexbox so that when adding responsiveness you can easily change the alignment from vertical to horizontal